releasenotes/notes/0.16/4989-Fixes-MCPhaseGate-definition-cbe5e14992116e67.yaml

```diff
@@ -0,0 +1,17 @@
+---
+fixes:
+  - |
+    Previously, when creating a multiply controlled phase gate via
+    :meth:`qiskit.circuit.library.PhaseGate.control`, an
+    :class:`~qiskit.circuit.library.MCU1Gate` gate had been incorrectly
+    returned. This has been fixed so that an
+    :class:`~qiskit.circuit.library.MCPhaseGate` is correctly returned now.
+  - |
+    Previously, attempting to call
+    :meth:`~qiskit.circuit.QuantumCircuit.decompose` on a
+    :class:`~qiskit.circuit.QuantumCircuit` object containing an
+    :class:`~qiskit.circuit.library.MCPhaseGate` would would raise an error due
+    to an inconsistency in the
+    :attr:`~qiskit.circuit.library.MCPhaseGate.definition` attribute. This issue
+    has been fixed and the
+    :attr:`~qiskit.circuit.library.MCPhaseGate.definition` attribute corrected.
```

Comment:
Backported in 0.15.2.

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/5142/commits/bdfce87f1dda045ec61dd740645aed055df041ee